### ISSUE TITLE
R1C1 formula translation doesn't handle double double-quotes

### ISSUE THREAD

**Issue by Thealexbarney** [EPPlusSoftware/EPPlus#252]
Setting an R1C1 formula containing a double double-quote will turn it into a single double-quote, causing Excel to consider the workbook corrupted.

```cs
using (var package = new ExcelPackage(new FileInfo("test.xlsx")))
{
    ExcelWorksheet ws = package.Workbook.Worksheets.Add("Sheet1");
    ws.Cells[1, 1].FormulaR1C1 = "\" \"\" \"";
    ws.Cells[1, 2].Formula = "\" \"\" \"";  // Sets the formula as expected
    package.Save();

    Console.WriteLine(ws.Cells[1, 2].FormulaR1C1);
    Console.WriteLine(ws.Cells[1, 2].Formula);

    // These both throw an "Unterminated string" exception
    Console.WriteLine(ws.Cells[1, 1].FormulaR1C1);
    Console.WriteLine(ws.Cells[1, 1].Formula);
}
```

**Comment by JanKallman** [comment]
Thanks for reporting it. Fix applied. Feel free to test it using our develop nuget feed - https://ci.appveyor.com/nuget/epplus.

**Comment by JanKallman** [comment]
Fixed in EPPlus 5.5
